components/mailchimp/sources/new-or-updated-subscriber/new-or-updated-subscriber.js

```diff
@@ -0,0 +1,50 @@
+const common = require("../common-webhook");
+const { mailchimp } = common.props;
+
+module.exports = {
+  ...common,
+  key: "mailchimp-new-or-updated-subscriber",
+  name: "New Or Updated Subscriber",
+  description:
+    "Emit an event when a subscriber is added or updated (on profile, or email address change) in an audience list.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    listId: {
+      type: "string",
+      label: "Audience List Id",
+      description:
+        "The unique ID of the audience list you'd like to watch for new and updated subscribers.",
+    },
+    server: { propDefinition: [mailchimp, "server"] },
+  },
+  methods: {
+    ...common.methods,
+    getEventName() {
+      return ["subscribe", "profile", "upemail"];
+    },
+    getEventType() {
+      return ["subscribe", "profile", "upemail"];
```

Comment:
No `unsubscribe` or `cleaned`?

Reply:
Unsure why were not included. Zapier has that same trigger but it doesn't specify which evets are supported.
I have included `unsubscribe `or cleaned as events to watch for when subscriber is updated.